etl/steps/data/garden/artificial_intelligence/2025-04-18/cset.meta.yml

```diff
@@ -0,0 +1,218 @@
+
+  description_short_citations: Citations take time to accumulate so recent years may have fewer citations, and the measure for these years may not completely reflective of their eventual total.
+
+  note_citations: The citation counts for papers published in a specific year are subject to a time lag, as papers often accrue most citations in subsequent years. Data for 2022-2023 is therefore incomplete.
+  note_articles:  Data for 2022-2023 is incomplete.
+  note_investment: Data is expressed in constant 2021 US$. Inflation adjustment is based on the US Consumer Price Index (CPI). Data for 2022-2023 is incomplete.
+  note_patents: According to calculations by CSET, the median time for a patent to be granted is 826 days from its initial filing date, while the average time is 860 days.  Data for 2021-2023 is incomplete.
+
+  description_key_articles: &description_key_articles |-
+    - CSET sources the data on the number of articles and citations from the Merged Academic Corpus (MAC), which includes over 260 million academic articles.
+    - Articles are marked as AI-related using an automated system.
+    - Articles are linked to countries based on the authors’ listed organizations—if at least one author is from an institution in a country, the article counts for that country.
+    - If authors are from multiple countries, the article counts once for each country, but only once per country even if there are multiple authors from the same place.
+    - For example, an article with authors from both the U.S. and Japan counts once for each.
+
+  description_key_patents:  &description_key_patents |-
+    - CSET sources patent data from 1790 Analytics, PATSTAT, and The Lens.
+    - Patent data can be delayed, as some patent offices take years to release filings, so the most recent years may be incomplete and should be used with caution.
+    - Here, we’ve chosen not to include the most recent years of data in the dataset, as they are likely incomplete, as noted in the original dataset provided by CSET.
+    - The dataset includes over 360,000 AI-related patent families—groups of documents related to a single invention—and counts each family as one patent, treating it as a "granted patent" if any document in the family has been granted, or as a "patent application" if all documents are still pending.
+    - Inventors often apply for the same patent in multiple countries, so the dataset groups all filings for one invention across jurisdictions into a single patent family, using the year and country of the first application to assign a filing date and location.
+    - For example, if someone files a robot patent first in the European Patent Office and then in China, all documents are grouped under one EPO patent.
+    - The dataset reports where patents are filed, not where inventors are from—for instance, many patents filed in the U.S. are by non-U.S. inventors—so filing location is used as a proxy for innovation and market interest.
```

Comment:
em dash

Reply:
And with spaces before and after in this case the one above https://www.notion.so/owid/Writing-and-style-guide-d51a3739ff8542ca90297fa8de40437c?pvs=4#81307e9aefb448208297e3fc173f7ef1